prac_05/state_names.py

```diff
@@ -11,11 +11,10 @@
 for pair in CODE_TO_NAME.items():
     print(f"{pair[0]:{MAX_SHORT_STATE_LENGTH}} is {pair[1]}")
```

Comment:
Does the abbreviated part in the variable MAX_SHORT_LENGTH follow this subjects variable naming conventions in the style guide?

Reply:
A separate note about MAX_SHORT_STATE_LENGTH

Could MAX_SHORT_STATE_LENGTH be renamed to be more consistent with the dictionary CODE_TO_NAME where they both refer to the keys in the dictionary?

Reply:
Both great points! I will change the constant name to MAXIMUM_CODE_LENGTH, to a) avoid confusion involving the built-in name 'max', and b) have consistent naming between the constant and dictionary.